build/tasks/clang-format.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+var gulp = require('gulp');
```

Comment:
Oh i didn't know that. Do you think we should? Is there anything that we are doing differently formatting wise? I think may be we should just in case we want modify options later on.